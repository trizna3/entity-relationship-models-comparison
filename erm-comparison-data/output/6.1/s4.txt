customers-users
auditoriums-halls
tickets-tickets
films-films
screenings-screenings
