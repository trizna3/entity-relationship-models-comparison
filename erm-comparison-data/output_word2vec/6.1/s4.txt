screenings-screenings
films-films
customers-users
tickets-tickets
auditoriums-halls
