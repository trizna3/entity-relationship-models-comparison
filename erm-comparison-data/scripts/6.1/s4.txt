entity set halls {
	"capacity"
};


entity set screenings {
	"date"
	"time"
};

entity set films {
	"title"
};

entity set tickets {
};

entity set users {
	"name"
	"email"
	"password"
};

association between 
{
	halls ONE 
	screenings MANY
}{};

association between 
{
	screenings MANY 
	films ONE
}{};

association between
{
	screenings ONE
	tickets MANY
}{};

association between 
{
	tickets ONE 
	users MANY 
}{};
