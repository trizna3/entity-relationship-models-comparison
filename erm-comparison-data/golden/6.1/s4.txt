auditoriums-halls
screenings-screenings
films-films
customers-users
tickets-tickets